assets/src/edit-story/components/panels/test/textStyle.js

```diff
@@ -273,11 +288,24 @@ describe('Panels/TextStyle', () => {
     });
 
     it('should update multi padding without lock and different padding', () => {
+      const { getByTestId, pushUpdateForObject } = renderTextStyle([
+        unlockPaddingTextElement,
+        unlockPaddingTextDifferentPadding,
+      ]);
+      const input = getByTestId('padding.horizontal');
+      fireEvent.change(input, { target: { value: '11' } });
+      expect(pushUpdateForObject).toHaveBeenCalledWith(
+        'padding',
+        { horizontal: 11 },
+        DEFAULT_PADDING
+      );
+    });
+
+    it('should default padding lock ration should be false when multiple elements with different lock ratio and update only horizontal', () => {
```

Comment:
Again, these descriptions are bit tricky (even with some spelling errors). Test descriptions are super important, because when they fail, someone else will have to understand what the purpose was 5 months from now!

* _should correctly update only vertical/horizontal padding when multiple elements with different padding lock settings are selected_

Reply:
Got it. Fixed! 👍